docs/ENTITY_RELATIONSHIP_DIAGRAM.md

```diff
@@ -0,0 +1,116 @@
+# PDC Schema
+## Entity Relationship Diagram
+
+```mermaid
+erDiagram
+  CanonicalField {
+    int id
+    string label
+    string type
+    datetime dateCreated
+  }
+  Applicant {
+    int id
+    string externalId 
+    bool hasGivenPermission
+  }
+  ExternalSource {
+    int id
+    string name
+  }
+  Application {
+    int id
+    int applicantId
+    int opportunityId
+    datetime dateCreated
+  }
+  Outcome {
+    int id
+    int applicationId
+    string outcome
+    datetime dateCreated
+  }
+  Opportunity {
+    int id
+    string title
+    datetime dateCreated
+  }
+  ApplicationSchema {
+    int id
+    int grantId
```

Comment:
Is this really `applicationID`?

Reply:
Yes! Thank you (and thank you @bickelj too)

I believe I've resolved it all now